easybuild/easyconfigs/r/R/R-3.6.3-foss-2020a.eb

```diff
@@ -0,0 +1,2321 @@
+    }),
+    ('class', '7.3-15', {
+        'checksums': ['f6bf33d610c726d58622b6cea78a808c7d6a317d02409d27c17741dfd1c730f4'],
+    }),
+    ('e1071', '1.7-3', {
+        'checksums': ['bb2dba526b673ec3a573befe365e3500b773593f0384fd6694e0835496bcc25d'],
+    }),
+    ('nnet', '7.3-13', {
+        'checksums': ['ae8820e292c7a413206ef82d1f41da2809c1d0e39c97f6800f9c7952599119fc'],
+    }),
+    ('minqa', '1.2.4', {
+        'checksums': ['cfa193a4a9c55cb08f3faf4ab09c11b70412523767f19894e4eafc6e94cccd0c'],
+    }),
+    ('RcppEigen', '0.3.3.7.0', {
+        'checksums': ['62ea627284425bfdb56613bc315cca492ed3483a56a03c1f9dc9821a25c3e8ac'],
+    }),
+    ('MatrixModels', '0.4-1', {
+        'checksums': ['fe878e401e697992a480cd146421c3a10fa331f6b37a51bac83b5c1119dcce33'],
+    }),
+    ('quantreg', '5.54', {
+        'checksums': ['703b2c67d88e95eaf1bf3c09fc64c1964bcc1145244c8bc8c0e5ac99713ad0a6'],
+    }),
+    ('robustbase', '0.93-5', {
+        'checksums': ['bde564dbd52f04ab32f9f2f9dd09b9578f3ccd2541cf5f8ff430da42a55e7f56'],
+    }),
+    ('zoo', '1.8-7', {
+        'checksums': ['9e072ddc8f245adcdeb230cbf0c818db9c028c320894f48211758da2bf2085f0'],
+    }),
+    ('lmtest', '0.9-37', {
+        'checksums': ['ddc929f94bf055974832fa4a20fdd0c1eb3a84ee11f716c287936f2141d5ca0a'],
+    }),
+    ('vcd', '1.4-6', {
+        'checksums': ['7f804081a722e98885c4bc8b57be67486ffa3dcb681a5ea3b2dcd0e26f225f66'],
+    }),
+    ('snowfall', '1.84-6.1', {
+        'checksums': ['5c446df3a931e522a8b138cf1fb7ca5815cc82fcf486dbac964dcbc0690e248d'],
+    }),
+    ('rpart', '4.1-15', {
+        'checksums': ['2b8ebe0e9e11592debff893f93f5a44a6765abd0bd956b0eb1f70e9394cfae5c'],
+    }),
+    ('survival', '3.1-11', {
+        'checksums': ['44f6120ff977ac576b8fa15c0626dfef8d80ba392dc03364b5e15dbb465d70b9'],
+    }),
+    ('bindr', '0.1.1', {
+        'checksums': ['7c785ca77ceb3ab9282148bcecf64d1857d35f5b800531d49483622fe67505d0'],
+    }),
+    ('plogr', '0.2.0', {
+        'checksums': ['0e63ba2e1f624005fe25c67cdd403636a912e063d682eca07f2f1d65e9870d29'],
+    }),
+    ('bindrcpp', '0.2.2', {
+        'checksums': ['48130709eba9d133679a0e959e49a7b14acbce4f47c1e15c4ab46bd9e48ae467'],
+    }),
+    ('purrr', '0.3.3', {
+        'checksums': ['0f31a89a424e12e35bd6e0581dee1d160f1f8be1f0a883a7d33ccbde8ef69e9e'],
+    }),
+    ('tidyselect', '1.0.0', {
+        'checksums': ['fe761766d03af86d04da9a9a7800e9c8271d2cb067776cfb817d853725d59caf'],
+    }),
+    ('dplyr', '0.8.5', {
+        'checksums': ['5750d3bf4bda7b5448e08af264ed183b4f7bd0c59a9d828fe9dd399b14590218'],
+    }),
+    ('tidyr', '1.0.2', {
+        'checksums': ['2403dd2f4d350d1ecb449be5d61cdccf6655572a807409325f9122c716924e69'],
+    }),
+    ('mnormt', '1.5-6', {
+        'checksums': ['2951fcc9711d577d7d5577d313875c4ed9f7a7a06df8c3b62fa27c231f021e4d'],
+    }),
+    ('foreign', '0.8-76', {
+        'checksums': ['052e580bc1e1334b41e7d693df341858395e72488d9c4b3ed8968bf04317ddfc'],
```

Comment:
You won't believe me, but the checksum for `foreign` `0.8-76` changed (since yesterday):

```diff
$ diff -ru foreign-0.8-76.old foreign-0.8-76
diff -ru foreign-0.8-76.old/DESCRIPTION foreign-0.8-76/DESCRIPTION
--- foreign-0.8-76.old/DESCRIPTION      2020-03-03 17:50:47.000000000 +0100
+++ foreign-0.8-76/DESCRIPTION  2020-03-03 17:51:03.000000000 +0100
@@ -51,4 +51,4 @@
   Free Software Foundation, Inc. [cph]
 Maintainer: R Core Team <R-core@R-project.org>
 Repository: CRAN
-Date/Publication: 2020-03-03 16:50:47 UTC
+Date/Publication: 2020-03-03 16:51:03 UTC
diff -ru foreign-0.8-76.old/MD5 foreign-0.8-76/MD5
--- foreign-0.8-76.old/MD5      2020-03-03 17:50:47.000000000 +0100
+++ foreign-0.8-76/MD5  2020-03-03 17:51:03.000000000 +0100
@@ -1,6 +1,6 @@
 f87d40597e85fd95afb45eb64c651832 *COPYRIGHTS
 9abc37ec459f59a8ca6bcbf8be819ca1 *ChangeLog
-2c5b458f9f053caed8c41cc8c9db367d *DESCRIPTION
+f1b6fbaacb2079c53ac976943025ff1d *DESCRIPTION
 2e82d3849af11b642299fc69ccef26f3 *GPL-2
 ed8178a14789e97ce12174176eed1908 *NAMESPACE
 a6c68bd0f84690aab0887b99db7077e0 *R/R_systat.R
```

```
$ ls -lrt foreign_0.8-76.tar.gz*
-rw-rw-r-- 1 vsc40023 vsc40023 346869 Mar 24 21:06 foreign_0.8-76.tar.gz.bk_20200324
-rw-rw-r-- 1 vsc40023 vsc40023 346884 Mar 25 15:43 foreign_0.8-76.tar.gz

$ sha256sum foreign_0.8-76.tar.gz*
8bdc9337345d9126e5882f0d1538eb714c3e39a40b78b53a73271adc8fa9625d  foreign_0.8-76.tar.gz
052e580bc1e1334b41e7d693df341858395e72488d9c4b3ed8968bf04317ddfc  foreign_0.8-76.tar.gz.bk_20200324
```

Reply:
fixed :+1: